tests/phpunit/tests/media.php

```diff
@@ -2265,7 +2265,6 @@ public function test_wp_filter_content_tags_srcset_sizes() {
 			$respimg_xhtml,
 			$respimg_html5
 		);
-		$content_filtered = wp_img_tag_add_decoding_attr( $content_filtered, 'the_content' );
```

Comment:
We should still ensure that anyone using this function is getting the correct markup, so we don't introduce back-compat breaks, even though the function is deprecated.

Reply:
<!-- highlight specific problems -->
**❗️issue:** @joemcgill

The problem with this is, keeping tests using deprecated functions will [intentionally](https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/includes/abstract-testcase.php#L582-L639) result in failures.

<img width="1411" alt="Screenshot 2023-08-17 at 10 13 15" src="https://github.com/WordPress/wordpress-develop/assets/2189187/938bfab7-fca2-4db1-bf3e-726f277378e5">

Reply:
Ah! The way to handle this is to add `@expectedDeprecated wp_img_tag_add_decoding_attr` to the docblock for those test methods. See: https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#annotations

Reply:
Thanks for the tip, @joemcgill. These aren't the annotations I use daily, so it's easy to blur them :).